README.md

```diff
@@ -36,3 +36,152 @@ The `storage` subproject contains code related to the key-value storage of the R
 
 File issues in our Public Jira Instance: [File a bug](https://rchain.atlassian.net/secure/CreateIssueDetails!init.jspa?pid=10105&issuetype=10103&versions=10012&components=10004&assignee=medha&summary=issue+created%20via+link)
 
+## Building and Running
+
+This document is for do-it-yourselfers, who want to try out _very_ alpha versions of the RChain software. For those lacking in masochistic tendencies, periodic releases are available via Docker at [our Docker storefront](https://store.docker.com/profiles/rchain).
+
+### Developer Quick-Start
+
+When working in a single project, scope all `sbt` commands to that project. The most effective way is to maintain a running `sbt` instance, invoked from the project root:
+```
+$ sbt
+[info] Loading settings from plugins.sbt ...
+[info] Loading global plugins from /home/kirkwood/.sbt/1.0/plugins
+[info] Loading settings from plugins.sbt,protoc.sbt ...
+[info] Loading project definition from /home/kirkwood/src/rchain/project
+[info] Loading settings from build.sbt ...
+[info] Set current project to rchain (in build file:/home/kirkwood/src/rchain/)
+[info] sbt server started at local:///home/kirkwood/.sbt/1.0/server/e6a65c30ec6e52272d3a/sock
+sbt:rchain> project storage
+[info] Set current project to storage (in build file:/home/kirkwood/src/rchain/)
+sbt:storage> compile
+[... compiling storage ...]
+```
+but single-line commands work, too:
+```
+$ sbt "project storage" clean compile test
+```
+or
+```
+$ sbt "storage / clean" "storage / compile" "storage / test"
+```
+
+### Building
+
+The build is organized into several, mostly autonomous projects. These projects may be built (and used!) on their own, or they may be combined together to form the full node package. The build process in any case is contained in and controlled by a single, top-level `build.sbt` file. This process is able to produce several different kinds of artifacts, including JAR files (for Scala) and Docker images.
+
+The most up-to-date code is found in the `dev` branch. This brilliant, cutting-edge source is periodically merged into `master`, which branch should represent a more stable, tested version.
+
+#### Subprojects
+
+The available subprojects may be listed by `sbt`:
+```
+sbt:rchain> projects
+[info] In file:/home/kirkwood/src/rchain/
+[info] 	   comm
+[info] 	   node
+[info] 	 * rchain
+[info] 	   rholang
+[info] 	   roscala
+[info] 	   roscala_macros
+[info] 	   storage
+```
+
+In most cases, simply building the project you care about is enough:
+
+```
+sbt:rchain> storage / compile
```

Comment:
Nitpick:

Slightly more common to see this written as:
```
sbt:rchain> storage/compile
```

Reply:
Huh. That hadn't been working for me, but it does now. I'll change it in some future pass.

Reply:
👍

Reply:
In fact, I'll do it in https://github.com/rchain/rchain/pull/302.